src/components/FileTabs/FileTabs.tsx

```diff
@@ -91,149 +92,124 @@ const useSelect = () => {
+    setList(
+      Children.map(children, (_, index) => ({
+        id: `tab-element-${index}`,
+      }))
+    );
   }, [children]);
-  const updateOrder = (order: number) => {
-    if (dragElement.current) {
-      const { value } = dragElement.current;
-      setOrderList((list: string[]) => {
-        const newList = list.filter(element => element !== value);
-        newList.splice(order, 0, value);
-        return newList;
-      });
-    }
-  };
 
-  const onClose = (value: string) => {
-    const index = orderList.findIndex(option => option === value);
-    onCloseProp(value, index);
-    setOrderList((list: string[]) => {
+  const onClose = (index: number) => {
+    onCloseProp(index);
+    setList(list => {
       list.splice(index, 1);
       return [...list];
     });
   };
 
-  const onSelect = (value: string) => {
-    const index = orderList.findIndex(option => option === value);
-    onSelectProp(value, index);
+  const onSelect = (index: number) => () => {
+    onSelectProp(index);
   };
 
   const value = {
-    dragElementContainer,
-    selected,
-    list: orderList,
-    updateOrder,
-    onSelect,
+    selectedIndex,
     onClose,
   };
 
-  const onDragEnd = (e: DragEvent) => {
-    e.preventDefault();
-    if (dragElement.current) {
-      setOrderList(getOrderFromChildren(children));
-      dragElement.current = null;
-    }
-    if (dragElementContainer.current) {
-      dragElementContainer.current.innerHTML = "";
-    }
-  };
-
-  const onDrop = () => {
-    if (dragElement.current) {
-      const { order, value } = dragElement.current;
-      dragElement.current = null;
-      const destinationPosition = orderList.findIndex(element => element === value);
-      onReorderTab({ sourcePosition: order, destinationPosition, sourceValue: value });
-    }
-  };
-
-  const onDragStart = (e: DragEvent) => {
-    const order = +e.dataTransfer.getData("order");
-    const value = e.dataTransfer.getData("value");
-    dragElement.current = {
-      order,
-      value,
-    };
-  };
-
   const onWheel = (evt: WheelEvent<HTMLDivElement>) => {
-    evt.currentTarget.scrollLeft += evt.deltaY;
-  };
-
-  const onDragLeave = (e: MouseEvent<HTMLDivElement>) => {
-    updateOrder(Children.count(children));
-    if (typeof onMouseLeaveProp === "function") {
-      onMouseLeaveProp(e);
+    if (ref.current) {
+      ref.current.scrollLeft += evt.deltaY;
     }
   };
-
   return (
     <TabContext.Provider value={value}>
       <TabsContainer
-        onDragStart={onDragStart}
-        onDragEnd={onDragEnd}
-        onDragLeave={onDragLeave}
-        onDrop={onDrop}
+        ref={ref}
         onWheel={onWheel}
         onScroll={e => {
           e.preventDefault();
           e.stopPropagation();
         }}
-        {...props}
       >
-        <DragElementContainer ref={dragElementContainer} />
-        {children}
+        <TabsSortableContainer
+          as={ReactSortable}
+          style={{ display: "flex" }}
```

Comment:
why not adding this css line with the rest of the component css?

Reply:
Forgot to remove it